### PR TITLE
docs: change primary to default

### PR DESCRIPTION
**Pull request by danieltprice**

### CODE REVIEW

File: src/components/pages/pricing/hero/hero.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const items = [
     description: 'Generous always-available free tier, no credit card required.',
     features: [
       { title: '0.5 GiB storage' },
-      { title: '24/7 for your main compute', info: 'Plus 20h of usage for branches' },
+      { title: '24/7 for your primary compute', info: 'Plus 20h of usage for branches' },
```
Comment (zaynetro) [inline src/components/pages/pricing/hero/hero.jsx:26]: `default`?

File: src/components/pages/pricing/plans/data/plans.json [hunk 2]
```diff
@@ -54,7 +54,7 @@
         "subtitle": "Compute hrs = CPU size x active hrs"
       },
       "free": {
-        "title": "24/7 for main compute",
+        "title": "24/7 for primary compute",
```
Comment (zaynetro) [inline src/components/pages/pricing/plans/data/plans.json:57]: `default`?

**Review by zaynetro — APPROVED** [review]
Shall we add a note somewhere mentioning that `default` branch used to be called `primary`?

File: src/components/pages/pricing/hero/hero.jsx (hunk 1 above)
Comment (danieltprice) [reply src/components/pages/pricing/hero/hero.jsx:26]: We're now going to have "Primary" and "Read Replica" computes, as per: https://github.com/neondatabase/cloud/issues/14509

File: src/components/pages/pricing/plans/data/plans.json (hunk 2 above)
Comment (danieltprice) [reply src/components/pages/pricing/plans/data/plans.json:57]: see above.

**Pull request merged**